examples/landing-zone-v2/tier3/dns/address.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# reserve an external IP address for a tier 3 DNS A record
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:
+  name: workload-name-external-ip # kpt-set: ${workload-name}-external-ip
+  namespace: project-id-tier3 # kpt-set: ${project-id}-tier3
```

Comment:
specifying a namespace is not required for tier3 nor tier4 as they are resources observed by repo-sync

Reply:
you can remove that key